database/models/core.py

```diff
@@ -133,6 +133,8 @@ class Repository(CodecovBaseModel):
         ),
     )
     language = Column(types.Text)
+    languages = Column(postgresql.ARRAY(types.String), nullable=True, default=[])
```

Comment:
Adheres to these https://github.com/codecov/codecov-api/blob/main/core/models.py#L106